out_writeapi.go

```diff
@@ -236,12 +261,10 @@ func FLBPluginInit(plugin unsafe.Pointer) int {
 		managedwriter.WithMaxInflightRequests(queueSize),
 	)
 	if err != nil {
-		log.Fatal("NewManagedStream: ", err)
+		log.Fatal("NewManagedStream failed in FLBPluginInit: ", err)
```

Comment:
This line won't terminate the program, right?

Reply:
It will. If we can't get the manages stream wouldn't we also want to terminate the program?